### ISSUE TITLE
Weird large icons in sidebar

### ISSUE THREAD

**Issue by wout** [elementary/photos#664]
### What Happened?

The sidebar is showing large icons:

![image](https://user-images.githubusercontent.com/107324/139231348-51b23cf3-7b1a-4a55-b0e3-448d0a8b3caa.png)

It looks like the stylesheet got messed up.

### Steps to Reproduce

1. Open Photos

### Expected Behavior

Icons should be small.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Hardware Info

![image](https://user-images.githubusercontent.com/107324/139231705-ee2dbd22-c525-4e68-a6e2-aba7b1f09ccb.png)

**Comment by wout** [comment]
This problem persists until today.

![image](https://user-images.githubusercontent.com/107324/185657239-71e3f894-c6a0-482f-b64c-5896b26710ba.png)

![image](https://user-images.githubusercontent.com/107324/185657405-e702fc1e-ddb1-4149-ad29-3dd8f89fae10.png)

**Comment by zeebok** [comment]
Are you running in HiDPI mode? That seems to be when I see the issue.

**Comment by wout** [comment]
@zeebok Yes, I am. But I've been in HiDPI mode for four years on elementary, and this issue only popped up last October. In the meantime, I did a clean install, and still, they are that way. So, it was introduced with some update.

**Comment by zeebok** [comment]
Cool I will try to take a look into this since I can replicate it

**Comment by wout** [comment]
This has become better with OS 7. But the icons are still positioned with an offset:

![image](https://user-images.githubusercontent.com/107324/227789664-c11e8880-cd96-4e41-977e-bb9efc7b6956.png)
